src/assets/v1.3.0/sign_message_lib.json

```diff
@@ -0,0 +1,66 @@
+{
+  "defaultAddress": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+  "contractName": "SignMessageLib",
+  "version": "1.3.0",
+  "released": true,
+  "networkAddresses": {
+    "1": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "4": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "42": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "5": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "56": "",
+    "100": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "137": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "246": "0xa25b3579a295be016de5eb5F082b54B12d45F72C",
+    "4002": "",
+    "42161": "",
```

Comment:
If not already in progress it would be ok to not deploy the contract to all of these networks